### DISCUSSION TITLE
arm镜像问题

### DISCUSSION THREAD

**Discussion by SupRenekton** [labring/sealos#4817] [Q&A] [answered]
我在国产的银河麒麟操作系统的机器上用sealos部署k8s集群报错了，sealos应该支持arm架构的机器上搭建k8s集群的吧，我使用的sealos版本是sealos_4.3.7_linux_arm64.tar.gz这个，然后下载的k8s版本是registry.cn-shanghai.aliyuncs.com/labring/kubernetes:v1.25.16，
但是报了WARNING：image platform(linux/amd64) does not match the expected platform(linux/arm64)的警告，同时报错：“/var/lib/sealos/data/default/rootfs/opt/sealctl 无法执行二进制文件：可执行文件格式错误”
这是啥问题啊，是镜像不支持arm架构？但为啥只是警告啊

**Comment by bxy4543** [comment, marked as answer]
你的镜像是amd64镜像，拉取镜像在amd64机器上的话需要执行 sealos pull --platform linux/arm64 xxx

**Reply by SupRenekton** [reply]
多谢，我试试
